### ISSUE TITLE
v1.1.3 Windows - An unknown error ocurred: -1 when unwrapping repeatedly and mesh is far away from scene origin

### ISSUE THREAD

**Issue by JohnPeppermint** [DanielBoxer/UVgami#10]
First off, UVgami is awesome, thank you for making the tool free and open source.

Bug report:

TLDR: If I'm using "preserve mesh - full" and X symmetry, then I add a cube, unwrap it with UVgami and then I move it 6 units to the right and try to unwrap it with UVgami again, the error occurs. (I'm using Windows v1.1.3 on Blender 4.1, if it helps).

Long version:

If I'm using "preserve mesh - full" and X symmetry together and try to unwrap a mesh with UVgami repeatedly, it doesn't do anything except throw an "An unknown error ocurred: -1". 

After trying to replicate the error it seems to happen when I'm trying to unwrap again with UVgami and the mesh is far away from location (0, 0, 0) of the scene, but even location (6, 0, 0) is enough for it to happen. I've noticed that the origin of the mesh gets moved to 0,0,0 after unwrapping and after that the error seems to occur.

Sometimes moving the mesh closer to 0,0,0 fixes it but not always. If I copy the mesh to another file and try again it sometimes fixes it. 

I hope it helps!

**Comment by danielboxer** [comment]
Thanks for the bug report!
